.github/workflows/test.yml

```diff
@@ -257,6 +257,60 @@ jobs:
           command: build
           args: --target=${{ matrix.rust_target }} --manifest-path=${{ matrix.manifest }}
 
+  cross-windows-build:
+    name: Cross-compile ${{ matrix.manifest }} for ${{ matrix.rust_target }} from x86_64-unknown-linux-gnu
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+      matrix:
+        rust_target:
+          - x86_64-pc-windows-msvc
+          - i686-pc-windows-msvc
+        manifest: ['psm/Cargo.toml', 'Cargo.toml']
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+      - name: Add toolchain shims
+        run: |
+          set -eux
+          sudo ln -s clang-12 /usr/bin/clang-cl
+          sudo ln -s llvm-ar-12 /usr/bin/llvm-lib
+          sudo ln -s lld-link-12 /usr/bin/lld-link
+      - name: Install Windows SDK
+        env:
+          xwin_version: "0.1.6"
```

Comment:
I would make it a `matrix` variable. Helps with both keeping most of the variables in one place and also in the future would require fewer changes if we wanted to add additional versions to test with.